monkey/infection_monkey/monkey.py

```diff
@@ -174,8 +174,8 @@ def _get_arguments(args):
     @staticmethod
     def _get_otp() -> OTP:
         # No need for a constant, this is a feature flag that will be removed.
-        if OTP_FLAG not in os.environ:
-            return OTP("PLACEHOLDER_OTP")
+        # if OTP_FLAG not in os.environ:
+        #     return OTP("PLACEHOLDER_OTP")
```

Comment:
Either we can leave this or remove it. I'm not sure this PR is the right place to remove it.

Reply:
Just a temporary commit for testing